tests/all/integration/api/validation/test_discount_codes.py

```diff
@@ -64,6 +64,49 @@ def test_quantity_max_gt_tickets_number(self):
         with self.assertRaises(UnprocessableEntity):
             DiscountCodeSchemaTicket.validate_quantity(schema, data, original_data)
 
+    def test_amount_lte_ticket_price(self):
+        """
+        Discount Code Validate Amount Value - Tests if function runs without an exception
+        :return:
+        """
+        with app.test_request_context():
+            ticket = TicketFactory()
+            ticket.price = 100
+            save_to_db(ticket)
+
+            schema = DiscountCodeSchemaTicket()
+            original_data = {
+                'data': {}
+            }
+            data = {
+                'type': 'amount',
+                'value': 70,
+                'tickets': ['1']
+            }
+            DiscountCodeSchemaTicket.validate_value(schema, data, original_data)
+
+    def test_amount_gt_ticket_price(self):
+        """
+        Discount Code Validate Amount Value - Tests if exception is raised when discount value is gt ticket price
+        :return:
+        """
+        with app.test_request_context():
+            ticket = TicketFactory()
+            ticket.price = 100
+            save_to_db(ticket)
+
+            schema = DiscountCodeSchemaTicket()
+            original_data = {
+                'data': {}
+            }
+            data = {
+                'type': 'amount',
+                'value': 150,
+                'tickets': ['1']
+            }
+            with self.assertRaises(UnprocessableEntity):
+                DiscountCodeSchemaTicket.validate_value(schema, data, original_data)
```

Comment:
No need of this line, is it not being already tested in the unit test below?

Reply:
No. Actually the their is separate test for `amount` and `percent` type. Unit test contains the test related to `amount` type

Reply:
Why can't percent test be Unit Test as well?